examples/sonar_cloud_metrics.py

```diff
@@ -22,21 +22,36 @@ def run(self, repos: List[Repo]) -> None:
         series = []
 
         for repo in repos:
-            measures = self.sonar_cloud_api.get_component_measures(repo.org, repo.name, METRICS)
+            for branch in self.get_branches(repo):
+                measures = self.sonar_cloud_api.get_component_measures(repo.org, repo.name, branch.name, METRICS)
 
-            if measures:
-                series += self.get_series(repo, measures)
+                if measures:
+                    series += self.get_series(repo, branch, measures)
 
-        print("Series data:", series)
+        print('Series data:', series)
 
         self.datadog_api.submit_metrics(series)
 
-    def get_series(self, repo: Repo, measures: List[Dict[str, Any]]) -> Iterator[Series]:
+    def get_branches(self, repo: Repo) -> List[ProjectBranch]:
+        branches = self.sonar_cloud_api.get_project_branches(repo.org, repo.name) or []
+
+        # Filter out non-main, non-pre-release branches.
+        main_branch = [branch for branch in branches if branch.is_main]
+        pre_release_branch = [branch for branch in branches if branch.is_pre_release]
+
+        if len(main_branch) > 1:
```

Comment:
Should we error out when this is 0 too?

Reply:
Not all repos are onboarded yet with SonarCloud so they'd show 0 branches.